source/ed/connectors/geopal_parser.cpp

```diff
@@ -68,9 +68,9 @@ void GeopalParser::fill() {
 
     this->fill_admins();
     LOG4CPLUS_INFO(logger, "Admin count: " << this->data.admins.size());
-    this->fill_nodes();
-    LOG4CPLUS_INFO(logger, "Noeud count: " << this->data.nodes.size());
+    this->fill_postal_codes();
     this->fill_ways_edges();
+    LOG4CPLUS_INFO(logger, "Noeud count: " << this->data.nodes.size());
```

Comment:
it's not your code, but since you modified the line can you change it to `Node count ?`